index.html

```diff
@@ -26,55 +26,127 @@ <h1 class="intro-section__title">I’m Arthur.
+        </ul>
+      <button class="button__std btn">See Project
+      </button>
+    </sub>
+    <sub class="project__card">
+      <img class="card__img" src="Assets/Snapshoot Portfoliop4.svg">
+      <h2 class="card__title">Multi-Post Stories
+      </h2>
+      <div class="card__details">
+        <h4 class="details__title">CANOPY</h4>
+        <img src="Assets/Counterball.svg" alt="separator">
+        <h4 class="details__text">Back End Dev</h4>
+        <img src="Assets/Counterball.svg" alt="separator">
+        <h4 class="details__text">2015</h4>
+      </div>
+      <h4 class="card__description">A daily selection of privately personalized reads; no accounts or sign-ups required.        
+      </h4>
+        <ul class="tag__parent">
+          <li class="tag">HTML</li>
+          <li class="tag">CSS</li>
+          <li class="tag">Javascript</li>
+        </ul>
+      <button class="button__std btn">See Project
       </button>
     </sub>
-
-
   </section>
 
+  <!-- About Me section -->
+
   <section class="about__section">
-    <h2>About Myself</h2>
-    <h3>
+    <h2 class="about__title">About Myself</h2>
+    <p class="about__subtitle">
       Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
-    </h3>
-    <h3 class="intro-section__capstext">LET’S CONNECT</p>
+    </p>
+    <h3 class="intro-section__capstext">LET’S CONNECT</h3>
       <ul class="intro-section__social">
-        <li><a href="https://linkedin.com"><img src="Assets/Linkedin iconlinkedin.svg" alt="Linkedin"></a></li>
-        <li><a href="https://twitter.com"><img src="Assets/twitter.svg" alt="Twitter"></a></li>
-        <li><a href="https://medium.com"><img src="Assets/Vectormedium.svg" alt="Medium"></a></li>
-        <li><a href="https://www.angellist.com/"><img src="Assets/Vectorangellist.svg" alt="AngelList"></a></li>
-        <li><a href="https://github.com"><img src="Assets/Vectorgithub.svg" alt="Github"></a></li>
+        <li><a class="social" href="https://linkedin.com"><img class="social" src="Assets/Linkedin iconlinkedin.svg" alt="Linkedin"></a></li>
+        <li><a class="social" href="https://twitter.com"><img class="social" src="Assets/twitter.svg" alt="Twitter"></a></li>
+        <li><a class="social" href="https://medium.com"><img class="social" src="Assets/Vectormedium.svg" alt="Medium"></a></li>
+        <li><a class="social" href="https://www.angellist.com/"><img class="social" src="Assets/Vectorangellist.svg" alt="AngelList"></a></li>
+        <li><a class="social" href="https://github.com"><img class="social" src="Assets/Vectorgithub.svg" alt="Github"></a></li>
       </ul>
-      <button class="button__std">Get my Resume
+    <div class="about__button--parent">
+      <button class="button__std btn">Get my Resume
       </button>
+    </div>
 
   </section>
```

Comment:
- [ ] Excellent job developing your About me section  🚀. However, please kindly ensure that you include the Languages, Frameworks, and Skills inside this section too 🤓.

- [ ] Also, please kindly ensure to apply the correct border-radius to the top-right corner of the about me section in order to stick as much as possible to the Figma design. 🤓🔧

This is the Figma design 👇🏽


![figma](https://user-images.githubusercontent.com/116701485/235282152-a6a1e5fc-3c92-43a5-9f70-b60616ea54e5.png)

This is your design 👇🏽(almost there 😅)


![design](https://user-images.githubusercontent.com/116701485/235282075-1bdb54f2-e21d-4933-b4fe-48c8dba9eb73.png)